osmosis/assetlist.json

```diff
@@ -3474,6 +3474,68 @@
         }
       ]
     },
+    {
```

Comment:
If adding to Osmosis' assetlist, please move to the bottom of the list of assets.